### ISSUE TITLE
[breakage] tweakers.net , notifications icon is not working with updated uBO "uBlock filters" list

### ISSUE THREAD

**Issue by gitthehubs** [uBlockOrigin/uAssets#8839]
### URL(s) where the issue occurs
`https://tweakers.net/`

### Describe the issue

At the site `https://tweakers.net/` there is an icon for accessing your notifications and that is not working anymore, when you press the icon your notifications list is not opened anymore. No errors in the uBO logger.

### Screenshot(s)

None

### Versions

- Browser/version: Firefox 87 x64 / Windows 10 x64 all updated.
- uBlock Origin version: 1.34.0

### Settings

Default with testing.

### Notes

After troubleshooting I narrowed it down to the uBO "uBlock filters" list. When I unchecked this filterlist, apply changes, refreshed the website and then the notifications icon is working again. When I check the uBO "uBlock filters" list again, apply changes, refresh the website and it is still working. When I update the uBO "uBlock filters" list, apply changes, refresh the website and now it is not working again. So the problem occurs with the uBO "uBlock filters" update from this date, now 2 april 2021.
When I check the original uBO "uBlock filters" list there is nt tweakers.net website in the list. When I update the uBO "uBlock filters" list and check the list now tweakers.net is added to the list.

Solution: remove website tweakers.net from the uBO "uBlock filters" list.

Question: how did the website tweakers.net ended up in the list.? :)

**Comment by gitthehubs** [comment]
It is working again with this new update. Thanks!

**Comment by uBlock-user** [comment]
Ads will now appear at `https://tweakers.net/reviews/8396/laptop-best-buy-guide-december-2020.html`

**Comment by mapx-** [comment]
https://github.com/uBlockOrigin/uAssets/commit/8daba4cdf96ab0ca20d4dee36b809ae9cdec51f2

**Comment by gitthehubs** [comment]
> 
> 
> Ads will now appear at `https://tweakers.net/reviews/8396/laptop-best-buy-guide-december-2020.html`

Looks like it is fixed by mapx.

tweakers.net works together with hardware.info, both owned by DPG Media and they share the same officespace, and on hardware.info they did the same kind of ads serving from their own domain. It looks like tweakers.net copied that from hardware.info. I had to made a list of cosmetic filters for hardware.info to hide those ads. But I just did a check on the filterlist and saw that hardware.info is on it too, I removed my cosmetic filters and with a short test I saw no ads again. Great! 😄
